plugins/meta/portmap/README.md

```diff
@@ -0,0 +1,111 @@
+## Port-mapping plugin
+
+This plugin will forward traffic from one or more ports on the host to the
+container. It expects to be run as a chained plugin.
+
+## Usage
+You should use this plugin as part of a network configuration list. It accepts
+the following configuration options:
+
+* `noSnat` - boolean, default false. If true, do not set up the SNAT chains
```

Comment:
What do you think about naming this field `snat` (default true) instead of `noSnat`? At the surface, it seems clearer to understand than `noSnat` false.